spec/datadog/appsec/contrib/http/faraday_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'datadog/appsec/spec_helper'
+require 'faraday'
+
+RSpec.describe 'AppSec Faraday integration' do
+  let(:waf_response_double) { double(match?: false) }
```

Comment:
Should it be `instance_double` of an actual class to avoid stubbing non-existent methods?

Reply:
good point, thanks!